hudi-spark-datasource/hudi-spark/src/main/scala/org/apache/spark/sql/hudi/command/procedures/ShowMetadataTableColumnStatsProcedure.scala

```diff
@@ -50,11 +48,11 @@ class ShowMetadataTableColumnStatsProcedure extends BaseProcedure with Procedure
   )
 
   private val OUTPUT_TYPE = new StructType(Array[StructField](
-    StructField("file_name", DataTypes.StringType, nullable = true, Metadata.empty),
-    StructField("column_name", DataTypes.StringType, nullable = true, Metadata.empty),
-    StructField("min_value", DataTypes.StringType, nullable = true, Metadata.empty),
-    StructField("max_value", DataTypes.StringType, nullable = true, Metadata.empty),
-    StructField("null_num", DataTypes.LongType, nullable = true, Metadata.empty)
+    StructField("File Name", DataTypes.StringType, nullable = true, Metadata.empty),
```

Comment:
We don't need to modify this since we always use underscore naming convention for `OUTPUT_TYPE` for procedures?